packages/holocron-dev-server/lib/utils/logs/index.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
im not sure every directory needs its own index.js this adds additional redirection when looking through code, unless it is part of a public api then i am not sure of the benefit.

`import something from './some-path/something'` tells me exactly where i need to go to look at that file. 
`import something from './some-path'` means i have to go to the index.js of './some-path' and see whats being exported and from where.

Reply:
@Francois-Esquire  has updated this so that you can point directly to the file